src/ResourceManager/ServiceBus/Commands.ServiceBus/Commands.ServiceBus.csproj

```diff
@@ -59,26 +59,14 @@
     <Compile Include="Cmdlets\AzureServiceBusCmdletBase.cs" />
     <Compile Include="Cmdlets\CheckNameAvailability\TestAzureServiceBusCheckNameAvailability.cs" />
     <Compile Include="Cmdlets\Namespace\GetAzureServiceBusNamespace.cs" />
-    <Compile Include="Cmdlets\Namespace\GetAzureServiceBusNamespaceAuthorizationRules.cs" />
```

Comment:
It looks like you just removed the references to the cmdlets and not the cmdlets themselves.

Reply:
deleted the class files